src/main.ts

```diff
@@ -31,10 +31,24 @@ const COMMANDS: Record<string, Command> = {
 
 const ALLOWED_ACTIONS = ['created'];
 
+const URL_REGEX = /[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_\+.~#?&//=]*)?/gi;
```

Comment:
Weirdly overcomplicated regex, we don't really see much non-standard URLs.

Reply:
I just got the first one on StackOverflow IIRC. I am not that good in RegEx, do you have an suggestion of one to replace?

Reply:
This seems reasonable enough for what we usually see:

```
^(https?:\/\/)?([\w\-]+\.)+[\w\-]{2,}$
```